bin/prometheus-varnish-exporter

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
-sleep 1
+sleep 5
```

Comment:
This is probably not needed at all as the `-no-exit` prevents an exit when the stats fail to load.